.github/workflows/inference.yaml

```diff
@@ -18,6 +18,8 @@ jobs:
     steps:
       - name: Run Model Inference
         uses: redhat-et/time-to-merge-tool@v1
+        with: 
+          MODE: 1
```

Comment:
Shouldn't this be set to 0 for inference mode?

Reply:
yeah we can set this to 0 as default

Reply:
Updated :+1: